app/src/main/java/org/schabi/newpipe/player/BasePlayer.java

```diff
@@ -532,6 +538,18 @@ public void onShuffleClicked() {
         if (simpleExoPlayer == null) return;
         simpleExoPlayer.setShuffleModeEnabled(!simpleExoPlayer.getShuffleModeEnabled());
     }
+    /*//////////////////////////////////////////////////////////////////////////
+    // Mute / Unmute
+    //////////////////////////////////////////////////////////////////////////*/
+
+    public void onMuteUnmuteButtonClicled() {
```

Comment:
Rename this here and in other players
```suggestion
    public void onMuteUnmuteButtonClicked() {
```